server/api/indexer/themoviedb/index.ts

```diff
@@ -308,6 +309,12 @@ class TheMovieDb extends ExternalAPI {
         }
       );
 
+      data.episodes = data.episodes.map((episode) => {
+        if (episode.still_path != null && episode.still_path !== '') {
+          episode.still_path = `https://image.tmdb.org/t/p/original/${episode.still_path}`;
+        }
+        return episode;
+      });
```

Comment:
Why did you made this change?
For [src/components/TvDetails/Season/index.tsx](https://github.com/Fallenbagel/jellyseerr/pull/899/files#diff-7e0cc926ffb49b2afd2a833796b93b46dde16af82fa17bebb5b3262fa5f45f23)?

Reply:
Makes sense.
You could write your if condition like this (and you shouldn't use loose equality `!=` or `==`).

```suggestion
      data.episodes = data.episodes.map((episode) => {
        if (episode.still_path) {
          episode.still_path = `https://image.tmdb.org/t/p/original/${episode.still_path}`;
        }
        return episode;
      });
```

Reply:
done